packages/flame/lib/src/components/component_set.dart

```diff
@@ -128,9 +128,11 @@ class ComponentSet extends QueryableOrderedSet<Component> {
   }
 
   /// Marks a list of components to be removed from the components list on the
-  /// next game tick.
-  void removeAll(Iterable<Component> components) {
+  /// next game tick. This will return the same list as sent in.
+  @override
+  Iterable<Component> removeAll(Iterable<Component> components) {
     _removeLater.addAll(components);
```

Comment:
Is `_removeLater` defined in this class?

Reply:
Yes? You can just press view file if you don't want to dig it up in the source code.

Reply:
Shame on me. Was looking into `component` itself :D sorry